3ds/ci-build-3ds.sh

```diff
@@ -5,10 +5,9 @@ cd 3ds
 make -j8 || exit 1
 
 # Bundle and prepare for release
-mkdir OpenSupaplex
 mkdir 3ds
-mkdir cias
-mv OpenSupaplex.3dsx 3ds/OpenSupaplex.3dsx
-mv OpenSupaplex.cia cias/OpenSupaplex.cia
-cp -R ../resources/* OpenSupaplex/
-zip -r OpenSupaplex-3ds.zip OpenSupaplex 3ds cias INSTALL.TXT
+mkdir 3ds/OpenSupaplex
+mkdir cia
+mv OpenSupaplex.3dsx 3ds/OpenSupaplex/OpenSupaplex.3dsx
+mv OpenSupaplex.cia cia/OpenSupaplex.cia
+zip -r OpenSupaplex-3ds.zip 3ds cia INSTALL.TXT
```

Comment:
No need for the `OpenSupaplex` if data is stored in `sdmc:/OpenSupaplex` as suggested in another comment.
```suggestion
mkdir 3ds
mkdir cia
mv OpenSupaplex.3dsx 3ds/OpenSupaplex.3dsx
mv OpenSupaplex.cia cia/OpenSupaplex.cia
zip -r OpenSupaplex-3ds.zip 3ds cia INSTALL.TXT
```